addons/message_whats_new_v2.15/manifest.json

```
@@ -0,0 +1,45 @@
+{
+  "api_version": "0.1",
+  "id": "message_whats_new_v2.15",
+  "name": "What's new message",
+  "type": "message",
+  "conditions": {
+    "min_client_version": "2.15.0",
+    "max_client_version": "2.15.9"
+  },
+  "javascript": {
+    "enable": "setDate.js"
+  },
+  "message": {
+    "date": 1675040400,
+    "id": "message_whats_new_v2.15",
+    "title": "Welcome to Mozilla VPN 2.15",
+    "subtitle": "You’re using the latest version of Mozilla VPN. Take a look at the newest features and updates:",
+    "badge": "whats_new",
+    "blocks": [
+      {
+        "id": "c_1",
+        "type": "ulist",
+        "content": [
+          {
+            "id": "l_1",
+            "content": "We will now recommend a few of the highest-performing server locations when you’re on the Select Location screen."
+          },
+          {
+            "id": "l_2",
+            "content": "We’ve updated the App Exclusions screen to make it easier to choose which apps to leave out of VPN protection."
```

```suggestion
            "content": "We’ve updated the App exclusions screen to make it easier to choose which apps to leave out of VPN protection."
```

This is not [how it's spelled in the app](https://github.com/mozilla-mobile/mozilla-vpn-client/blob/main/translations/strings.yaml#L304). 

I think I've reported this issue in review a few times at this point, can we make a note about it (@chrisrooneys)?